<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesForm</class>
 <widget class="QWidget" name="PreferencesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>555</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>841</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="species">
      <property name="title">
       <string>Species</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="dogCheckBox">
         <property name="text">
          <string>Dog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="catCheckBox">
         <property name="text">
          <string>Cat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="rabbitCheckBox">
         <property name="text">
          <string>Rabbit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="rodentCheckBox">
         <property name="text">
          <string>Rodent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fishCheckBox">
         <property name="text">
          <string>Fish</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="birdCheckBox">
         <property name="text">
          <string>Bird</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="temperament">
      <property name="title">
       <string>Temperament</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QCheckBox" name="happyCheckBox">
         <property name="text">
          <string>Happy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="friendlyCheckBox">
         <property name="text">
          <string>Friendly</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="introvertedCheckBox">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Introverted</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="activeCheckBox">
         <property name="text">
          <string>Active</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="vigilantCheckBox">
         <property name="text">
          <string>Vigilant</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="goodWith">
      <property name="title">
       <string>Good With</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QCheckBox" name="kidsCheckBox">
         <property name="text">
          <string>Kids</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="animalsCheckBox">
         <property name="text">
          <string>Animals</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="shelter">
      <property name="title">
       <string>Shelter</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="bestFriendsCheckBox">
         <property name="text">
          <string>Best Friends</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="humaneSocietyCheckBox">
         <property name="text">
          <string>Humane Society</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="animalWelfareLeagueCheckBox">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Animal Welfare League</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="gender">
        <property name="title">
         <string>Gender</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="maleCheckBox">
           <property name="text">
            <string>Male</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="femaleCheckBox">
           <property name="text">
            <string>Female</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="age">
        <property name="title">
         <string>Age</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="youngCheckBox">
           <property name="text">
            <string>Young</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="adultCheckBox">
           <property name="text">
            <string>Adult</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="seniorCheckBox">
           <property name="text">
            <string>Senior</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="size">
        <property name="title">
         <string>Size</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QCheckBox" name="smallCheckBox">
           <property name="text">
            <string>Small</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="mediumCheckBox">
           <property name="text">
            <string>Medium</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="largeCheckBox">
           <property name="text">
            <string>Large</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Take a Fun Quiz Instead!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>My Preferences</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
